requirements.txt

```diff
@@ -1,7 +1,7 @@
 Flask==0.10.1
 Flask-Admin==1.0.8
 Flask-SQLAlchemy==1.0
-GitHub-Flask==0.3.4
+GitHub-Flask==2.0.1
```

Comment:
There isn't a changelog unfortunately but it doesn't look like that much has changed: https://github.com/cenkalti/github-flask/compare/7197116b27fe6b2ba694e49d6d2b54698b5e1505...master

Actually it's dropped the requirement for the `GITHUB_CALLBACK_URL` variable so I'll remove it from the env